src/client/components/sideMenubar/sideMenu.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './sideMenu.styles.css';
+import homeImg from '../../assets/images/home_sidemenu.png';
+import nextImg from '../../assets/images/nextsteps_sidemenu.png';
+import elevatorImg from '../../assets/images/elevator_sidemenu.png';
+
+function SideMenu() {
+  return (
+    <div className="side_menu_div">
+      <nav>
+        <ul className="side_menu_ul">
+          <li className="side_menu_whiteborder home">
+            <img src={homeImg} alt="homeImg" />
+          </li>
+          <li className="side_menu_spacebetween">
+            <a href="#dummylink">Home</a>
+          </li>
+          <li className="side_menu_whiteborder">
+            <img src={nextImg} alt="nextImg" />
+          </li>
+          <li className="side_menu_spacebetween">
+            <a href="#dummylink">Next steps</a>
+          </li>
+          <li className="side_menu_whiteborder">
+            <img src={elevatorImg} alt="elevatorImg" />
+          </li>
+          <li className="side_menu_spacebetween">
+            <a href="#dummylink">Elevetor pitch</a>
+          </li>
+        </ul>
```

Comment:
The little indicator bar is not working as intended. Currently it points to the icon that the mouse is hovering over. The indicator should match the page it is displaying. To do this, you need to add conditional logic into your .js file. In simple words - if "Page Title" is equal to what the icon represents, then indicator bar goes next to icon.